test/ui-test/src/drivers/SquishDriverVerification.py

```diff
@@ -117,8 +117,8 @@ def verify_or_create_screenshot(vp: str, obj: Dict[str, Any]):
 def verify_screenshot(vp: str):
     test.vp(vp)
     
-def imagePresent(imageName: str, tolerant: bool = True, threshold: int = 99.5, minScale: int = 50, maxScale: int =  200, multiscale: bool = True):
-    test.imagePresent(imageName, {"tolerant": tolerant, "threshold": threshold, "minScale": minScale, "maxScale": maxScale, "multiscale": multiscale})
+def imagePresent(imageName: str, tolerant: bool = True, threshold: int = 99.5, minScale: int = 50, maxScale: int =  200, multiscale: bool = True, searchRegion: Any = "all"):
```

Comment:
I know it was already defined like this but could you refactor this method name to follow `lower_case_with_underscores`? We are trying to unify it.

Reply:
Good point. Done